src/Cache.php

```diff
@@ -313,4 +340,68 @@ public function getOrSet($key, callable $callable, $ttl = null, Dependency $depe
 
         return $value;
     }
+
+    public function __call($name, $arguments)
```

Comment:
What's the purpose for it?

Reply:
Some cache backends can have specific public methods (for example, `Memcached` implementation has `addServer()` method) we can't call that method if we use decorator with `Memcached` class.

Reply:
These methods are not meant to work runtime. There are more about configuration intended to be called before decorator is applied:

```php
$memcacheBackend = new Memcached();
$memcacheBackend->addServer(...);
$cache = new Cache($memcacheBackend);
```

I see no need to proxy these. Let's remove `__call()`.

Reply:
Yes, it's only for configuration. I'll remove it from `Cache`. 
But I'm thinking of adding `__call()` to the `Memcached` class. PHP `\Memcached` library has several methods related to configuration:
```
addServer
addServers
getOption
getServerByKey
getServerList
getStats
getVersion
isPersistent
isPristine
resetServerList
setOption
setOptions
setSaslAuthData
```
So, with `__call` method we don't need to add proxy method for each of those. What do you think?

Reply:
Another option is to just expose `\Memcached` instance via getter `getCache`.

Reply:
I don't like magic.

Reply:
What about getter?

Reply:
That's OK.

Reply:
Please do not use `__call()` when you can avoid it - it just adds another layer of magic and makes code harder to understand and debug.